viz.yaml

```diff
@@ -38,7 +38,9 @@ info:
     sf:
       repo: CRAN
       version: 0.6.0
-      
+    rjson:
```

Comment:
I've never heard of `rjson`! I've only used `jsonlite`.

Reply:
Actually....I think jsonlite is preferred....old habits die hard...I'll switch over